docs/manage/CLI.md

```diff
@@ -152,6 +159,43 @@ To switch to using an organization API key for authentication right away, [logou
 
 An organization can have multiple API keys.
 
+### Create a location API key
+
+To use an location API key to authenticate your CLI session, you must first create one:
+
+1. First, [authenticate](#authenticate) your CLI session.
+   If you don't already have a location API key created, authenticate using a personal access token or an organization API key.
+
+1. Then, run the following command to create a new location API key:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam locations api-key create --location-id <location-id>
+    --org-id <org-id> --name <key-name>
+   ```
+
+   Where:
+
+   - `location-id` is your location ID.
+     You can find your location ID by running `viam locations list` or by visiting your [robot fleet's page](https://app.viam.com/robots) in the Viam app.
+   - `org-id` is an optional organization ID to attach the key to.
+     You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam app](https://app.viam.com/).
+     If omitted, the key will attempt to attach itself to the organization of the location if there is only one organization associated with that location.
```

Comment:
when you send this for SME review, please ask what `attempt` means here. Shouldn't it just be that it does do that?

Reply:
Per offline conversation with @RoxyFarhad I added "If there are multiple organizations associated with that location, the operation will fail and you will need to specify the `org-id` explicitly."